source/minishift/index.html.erb

```diff
@@ -0,0 +1,183 @@
+---
+title: Minishift - Containerized OpenShift Cluster
+---
+
+<% content_for :header do %>
+<h1 class="animated fadeInDown delay">
+  Minishift
+</h1>
+<p class="animated fadeInUp delay">Develop Applications in a Local Containerized OpenShift Cluster</p>
+<% end %>
+
+<% content_for :navbar do %>
+<div class="nav_logo animated fadeIn">
+  <div class="container">
+    <div class="row">
+      <nav class="col-md-12">
+        <ul class="sf-menu" id="menu">
+          <li>
+            <a href="#about">ABOUT</a>
+          </li>
+          <li>
+            <a href="#getting-started">GET STARTED</a>
+          </li>
+          <li>
+            <a href="#resources">RESOURCES</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+  </div>
+</div>
+<% end %>
+
+<section class="container" id="about">
+  <div class="info_title info_vertical">
+    <div class="row wow info_vertical animated fadeInDown">
+      <div class="col-md-12 text-left">
+        <p>
+          Minishift is a tool that helps you run OpenShift locally by launching a single-node
+          OpenShift cluster inside a virtual machine. With Minishift you can try out OpenShift or
+          develop with it, day-to-day, on your local machine.
+        </p>
+        <p>
+          Minishift uses <a href="https://github.com/docker/machine/tree/master/libmachine">libmachine</a>
+          for provisioning VMs, and <a href="https://github.com/openshift/origin">OpenShift Origin</a>
+          for running the cluster.
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="container" id="getting-started">
+    <div class="vertical_line">
+      <div class="circle_bottom"></div>
+    </div>
+    <div class="row wow info_vertical animated fadeInUp">
+        <h1>Get Started</h1>
+        <div class="col-md-12 text-left">
+        The following high-level summary describes how to install and set up Minishift.
+        For detailed installation instructions, see the <a href="https://github.com/minishift/minishift/blob/master/README.md#installation">installation document</a>.
```

Comment:
That seems odd to me. You are saying this is a high level summary and that I should refer for more instructions I should refer to the README. At the same time, what follows below is exactly the content of the README. Can we not just point to the README here. At least then we have only one place to maintain and keep the installation instructions up to date. Eventually this might then change to the docs in docs.openshift.org, but for now having the README as single source seems to me to be the best approach.